behaviours/smarc_bt/smarc_bt/bt/conditions.py

```diff
@@ -129,4 +127,53 @@ def update(self) -> Status:
         self.feedback_message = f"({plan.seconds_to_timeout}) to timeout"
         if plan.timeout_reached: 
             return Status.FAILURE
-        return Status.SUCCESS
\ No newline at end of file
+        return Status.SUCCESS
+    
+class C_TaskIsMoveTo(Behaviour):
```

Comment:
Why not take the task-name as parameter in the constructor? :)
Then you can create `C_TaskIs(wara_ps_task_handler, "move-to")` and `C_TaskIs(wara_ps_task_handler, "move-path")` etc. with the same piece of code. 
Also include the task_name param in the name, `name=f'{self.__class__.__name__} {task_name}'` ish so we can see what task each version of this class is checking in the BT visualizer